terms.html

```diff
@@ -116,7 +116,7 @@
   <dt><dfn data-lt="named graphs">named graph</dfn></dt>
   <dd>
 A <a>graph</a> associated with specific properties, such as
-`verifiableCredential` or `proof`. These properties
+`verifiableCredential`. These properties
```

Comment:
@iherman this is removing all notions that `proof` is a separate graph, which doesn't seem like a good idea. You added this text, thoughts?

Reply:
Good point but, with this new setup, the fact that a `proof` create a new named graph must be added somewhere to the DI spec, and it is ok not to have it there.

At this point we probably should raise a new issue in the DI repo to remind us that this MUST be done, and leave this change as is.

Reply:
I agree with @iherman that this information belongs in DI - not VCDM.